.travis.yml

```diff
@@ -31,6 +31,8 @@ install:
   - sed -i -e 's|\$server_domain = "openfoodfacts.org";|\$server_domain = "off.travis-ci.org";|g' $TRAVIS_BUILD_DIR/lib/ProductOpener/Config2.pm
   - sed -i -e 's|\/home\/off|'$TRAVIS_BUILD_DIR'|g' $TRAVIS_BUILD_DIR/lib/ProductOpener/Config2.pm
 script:
+  - scripts/check_po_file.pl $TRAVIS_BUILD_DIR/po/common/en.po
```

Comment:
Do you think this could be added to `package.json` instead, so that everyone could run it with `npm test` independently of Travis CI?

Reply:
Probably, I'm really not familiar with travias and npm test, if you know how to do it, can you do it?

Reply:
`npm test` also just runs some scripts, but those are configured in package.json, so that everyone could run the complete "test suite" (Perl syntax, prove -l, check_po_file.pl, etc.) by just running `npm test`. I'll have a look. 😄